src/backend/app.js

```diff
@@ -21,10 +23,82 @@ app.use(cors());
 
 router.use("/meals", mealsRouter);
 
+// app.get("/my-route", (req, res) => {
+//   res.send("Hi friend");
+// });
+
+// /future-meals	Respond with all meals in the future (relative to the when datetime)
+
+app.get("/future-meals", async (req, res) => {
```

Comment:
These paths should be moved to file api/meals.js and then router should be used to access apis. 
Path to access api would become `http://localhost:5000/api/meals/future-meals` and that is more correct api path. We will explain these routers in next class